docs/topics/Gateway.md

```diff
@@ -701,6 +733,31 @@ A user's presence is their current state on a guild. This event is sent when a u
 }
 ```
 
+With Rich Presence
```

Comment:
Can we also change this to a table heading, like:

`### Example Game with Rich Presence`